setup.cfg

```diff
@@ -39,6 +39,7 @@ install_requires =
     pandas>1.0
     marshmallow>=3.7.1
     numpy>=1.18
+    whylabs-client >=0.1, <= 0.2
```

Comment:
I think the spaces here are causing flake8 to fail FYI

Reply:
idk i reran the jobs and everything passed. Looked like a random test failure 🤷